buildout_1.rst

```diff
@@ -16,9 +16,9 @@ Topics covered:
 
 .. only:: not presentation
 
-    `Buildout <https://pypi.python.org/pypi/zc.buildout>`_ composes your application for you, according to your rules.
+    `Buildout <https://pypi.python.org/pypi/zc.buildout>`_ configures your application for you, according to your rules.
 
-    To compose your application you must define the eggs you need, which version, what configuration files Buildout has to generate for you, what to download and compile, and so on.
+    To configure your application you must define the eggs you need, which version, what configuration files Buildout has to generate for you, what to download and compile, and so on.
     Buildout downloads the eggs you requested and resolves all dependencies. You might need five different eggs, but in the end, Buildout has to install 300 eggs, all with the correct version in order to resolve all the dependencies.
```

Comment:
I feel slightly bad because I could not convince you that compose is a better word. 
In the end, I cannot claim ownership for chapters I wrote and veto any changes I don't like.
The wording is important and it should be worded in such a way that the majority of the trainers feel comfortable with it and that it stays readable.
Because of that I'd love to hear opinions from @gomez @smcmahon @pbauer @macagua @fredvd @frapell @thet @giacomos